File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Enter a number:\n");
    }
    while (height < 1 || height > 8);
    // loop through the heights
    for (int i = 1; i <= height; i++)
    {
        //loop through each row
        for (int j = 1; j <= height; j++)
        {
            if (i + j <= height) //spaces initially depending on i
            {
                printf(" ");
            }
            else
            {
                printf("#"); // #'s start appearing then.
            }

        }

        printf("\n"); //new line after each row.


    }
}

File: pset6/mario/mario.py
from cs50 import get_int
# get a default int
n = -1
# get the int
while n < 1 or n > 8:
    n = get_int("Enter a number: ")
# start printing
for i in range(1, n + 1):
    print(' ' * (n - i) + '#' * i + '  ' + '#' * i )

